build.sh

```diff
@@ -1,4 +1,4 @@
-#!zsh
```

Comment:
My build scripts for MacOS now routinely rely on features of ZSH, which is now the Mac-standard shell. I don't guarantee that things will build on the Mac if you try to switch back to using Bash.